### PR TITLE
Repair the Absolutepath

### PR DESCRIPTION
**Pull request by ghost**
Addresses: https://github.com/dotnet/cli/issues/10293

### CODE REVIEW

File: scripts/obtain/dotnet-install.ps1 [hunk 1]
```diff
@@ -468,17 +468,21 @@ function Extract-Dotnet-Package([string]$ZipPath, [string]$OutPath) {
     }
 }
 
-function DownloadFile([Uri]$Uri, [string]$OutPath) {
+function DownloadFile([string]$Uri, [string]$OutPath) {
```
Comment (wli3) [inline scripts/obtain/dotnet-install.ps1]: rename $Uri to "Source" or different

File: scripts/obtain/dotnet-install.ps1 [hunk 2]
```diff
@@ -468,17 +468,21 @@ function Extract-Dotnet-Package([string]$ZipPath, [string]$OutPath) {
...
     if ($Uri -notlike "http*") {
-        Say-Verbose "Copying file from $Uri to $OutPath"
-        Copy-Item $Uri.AbsolutePath $OutPath
+        if (![System.IO.Path]::IsPathRooted($Uri)) {
+            $Uri = $(Join-Path -Path $pwd -ChildPath $Uri)
```
Comment (wli3) [inline scripts/obtain/dotnet-install.ps1]: Add comment why System.IO.Path.GetFullPath won't work

**Review by wli3 — APPROVED** [review]

File: scripts/obtain/dotnet-install.ps1 (hunk 1 above)
Comment (johnbeisner) [reply scripts/obtain/dotnet-install.ps1]: Done

**Comment by jerickmsft** [comment]
Um this just broke the script as the callers of DownloadFile are still calling with -Uri

**Comment by livarcocc** [comment]
@johnbeisner can you investigate?

**Comment by johnbeisner** [comment]
@livarcocc already on it...

**Pull request merged**